src/RefractAPI.cc

```diff
@@ -323,13 +323,17 @@ namespace drafter {
         // ideal solution is add warning into collection
         // but there is no way how to do it
         // in current time we solve it by rethrow
-
         catch (snowcrash::Error& e) {
-            e.location = payload.sourceMap->sourceMap;
-            throw e;
+            context.warnings.push_back(
```

Comment:
I feel like these warnings should have message. `'unable to render .... content.' + e.message`